tools/mcsema_disass/dyninst/MagicSection.cpp

```diff
@@ -0,0 +1,62 @@
+#include "MagicSection.h"
+
+#include "ExternalFunctionManager.h"
+
+#include <glog/logging.h>
+
+mcsema::ExternalVariable *MagicSection::WriteExternalVariable(
+    mcsema::Module &module,
+		const std::string &name) {
+  CHECK(start_ea) << "Magic section cannot start with 0!";
+
+	Dyninst::Address unreal_ea = AllocSpace(ptr_byte_size);
+
+	LOG(INFO) << "External var " << name
+						<< " is in magic_section at " << unreal_ea;
+	auto external_var = module.add_external_vars();
+	external_var->set_name(name);
+	external_var->set_ea(unreal_ea);
+
+	external_var->set_size(ptr_byte_size);
+
+	//TODO(lukas): This needs some checks
+	external_var->set_is_weak(false);
```

Comment:
In the README.md for this disass tool we should have a checklist of features that are an are not supported, so that it's easier to keep track of these kinds of things.